terraform/aks/config/preproduction.tfvars.json

```diff
@@ -0,0 +1,20 @@
+{
+  "cluster": "test",
+  "cluster_rg": "s189t01-tsc-pd-rg",
+  "cluster_name": "s189t01-tsc-test-aks",
+  "namespace": "tra-test",
+  "environment": "preproduction",
+  "deploy_azure_backing_services": false,
+  "enable_postgres_ssl": false,
```

Comment:
i don't think this will be doing anything, so you might as well remove it.
i.e It requires setting in the database.yml lik https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/config/database.yml#L26
You would only want prefer/false for review apps as they don’t use an azure Postgres service.